From eb6aac57b8bfaa4fdda8220bcc289a2e622f37d1 Mon Sep 17 00:00:00 2001
From: Serhii Kupriienko <skupriienko@anaconda.com>
Date: Mon, 12 Apr 2021 14:45:00 +0300
Subject: [PATCH] fixed gperf in CMakeList.txt

---
 CMakeLists.txt | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index e3840ae5..04dac4d8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -80,10 +80,6 @@ include(CheckStructHasMember)
 include(CheckPrototypeDefinition)
 
 find_package(Perl)
-find_program(GPERF NAMES gperf)
-if (NOT GPERF)
-    message(FATAL_ERROR "gperf not found!")
-endif()
 
 macro(config_write str)
 	file(APPEND "${CMAKE_BINARY_DIR}/include/config.h.in" "${str}")
-- 
2.23.4

